test/Microsoft.ML.AutoML.Tests/AutoMLExperimentTests.cs

```diff
@@ -223,6 +224,31 @@ public async Task AutoMLExperiment_UCI_Adult_Train_Test_Split_Test()
             result.Metric.Should().BeGreaterThan(0.8);
         }
 
+        [Fact]
+        public async Task AutoMLExperiment_UCI_Adult_Fairlearn_Test()
+        {
+            var context = new MLContext(1);
+            context.Log += (o, e) =>
+            {
+                if (e.Source.StartsWith("AutoMLExperiment"))
+                {
+                    this.Output.WriteLine(e.RawMessage);
+                }
+            };
+            var data = DatasetUtil.GetUciAdultDataView();
+            var experiment = context.Auto().CreateExperiment();
+            var pipeline = context.Auto().Featurizer(data, "_Features_", excludeColumns: new[] { DatasetUtil.UciAdultLabel })
+                                .Append(context.Auto().BinaryClassification(DatasetUtil.UciAdultLabel, "_Features_", exampleWeightColumnName: "signedWeight", useLgbm: false, useSdca: false, useLbfgs: false));
```

Comment:
@LittleLittleCloud Its failing here because there isn't a parameter named `useSdca` in the `BinaryClassification`.